src/components/Breadcrumbs/index.tsx

```diff
@@ -12,22 +12,28 @@ type BreadcrumbItem = {
 type Props = {
   route: string;
   platform: string;
-  isGen2: boolean;
 };
 
 const overrides = {
   '/': 'Home',
   '/javascript/prev': 'V6 (Preview)',
   '/swift/prev': 'V1',
   '/android/prev': 'V1',
-  '/flutter/prev': 'V0'
+  '/flutter/prev': 'V0',
+  '/javascript': 'Javascript',
+  '/react': 'React',
+  '/flutter': 'Flutter',
+  '/swift': 'Swift',
+  '/android': 'Android',
+  '/react-native': 'React Native',
+  '/angular': 'Angular',
+  '/nextjs': 'Next.js',
+  '/vue': 'Vue'
```

Comment:
Decided to just use the overrides here to replace "Overview" with the platform names, could also use logic to use the platform name instead of the page title, but this feels a little cleaner to me.

Reply:
Non-blocking, but do we have the platform names as constants somewhere? I don't want us having to make spelling updates in multiple places.

Reply:
Good callout, yes updated to use the constants